src/components/Button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Button.css';
+import { Props } from './types';
```

Comment:
1. Типы лучше импортировать с ключевым словом `type`. Их становится явно видно при чтении кода и при компиляции не будет происходить импорта модуля - TS просто спилит эту строку.

```suggestion
import type { Props } from './types';
```

2. Название `Props` подошло бы для описания типа пропсов, но не для типа возвращаемого значения
3. Не стоит использовать `React.FC` в виду [некоторых проблем](https://stackoverflow.com/questions/59988667/typescript-react-fcprops-confusion). А еще потому что он [deprecated](https://stackoverflow.com/questions/44133420/what-is-the-typescript-return-type-of-a-react-stateless-component) с момента релиза хуков.

В компонентах достаточно просто задавать тип для пропсов:

```Typescript
const Button = (props: Props) => {...}
```

Reply:
В [практикуме](https://praktikum.yandex.ru/learn/middle-frontend/courses/8bf82b38-54ba-4c62-bd31-6a4b373c26d6/sprints/4522/topics/1dc229bb-5aee-4ed5-827e-0022630ae654/lessons/0e066267-27c7-4e59-bba6-1074c307a573/) пишут использовать `FC` для функциональных компонентов

Reply:
Косяк в теории. Передам авторам, чтобы поправили